Makefile

```diff
@@ -36,7 +36,8 @@ docs-clean:
 	@cd docs; $(MAKE) clean
 
 lint-flake8:
-	flake8 . --ignore D203 --exclude docs/_build
+    # E501 and F401 are ignored because Pylint performs similar checks.
```

Comment:
Move this comment up one line.

Also, note that make file syntax requires that make recipes (like this line of code) be preceded by one or more tabs, *not* spaces.